lua/starfall/libs_cl/render.lua

```diff
@@ -22,6 +22,7 @@ registerprivilege("render.fog", "Render Fog", "Allows the user to control fog",
 local cv_max_fonts = CreateConVar("sf_render_maxfonts", "30", { FCVAR_ARCHIVE })
 local cv_max_rendertargets = CreateConVar("sf_render_maxrendertargets", "20", { FCVAR_ARCHIVE })
 local cv_max_maxrenderviewsperframe = CreateConVar("sf_render_maxrenderviewsperframe", "2", { FCVAR_ARCHIVE })
+local cv_max_pixelhandles = CreateConVar("sf_render_maxrenderviewsperframe", "50", { FCVAR_ARCHIVE })
```

Comment:
This convar should be sf_render_maxpixvishandlesperframe

Reply:
Oops... and I swear I had fixed that? I did try starting from scratch a few times while struggling with the code last night, maybe I accidentally undid it.